CONTRIBUTING.md

```diff
@@ -44,9 +41,9 @@ which come about as `l3build`'s focus is testing and building LaTeX packages:
 
 If you are submitting a pull request, notice that
 
-- We use Travis-CI for (light) testing so add `[ci skip]` to documentation-only
-  commit messages
-- We favour a single linear history so will rebase agreed pull requests on to
-  the `master` branch
+- We use GitHub Actions for (light) testing so add `[ci skip]` to
```

Comment:
`[ci skip]` isn't really needed any more, particularly as the GH Actions setup means we typeset the docs and find errors there independent of the code.